schema/2.0.0/corpora_schema.md

```diff
@@ -664,12 +674,32 @@ When a dataset is uploaded, the cellxgene Data Portal MUST automatically add the
     </tr>
     <tr>
       <th>Value</th>
-        <td><code>str</code>. This MUST be the human-readable ENSEMBL gene name assigned to the <code>feature_id</code>.  
+        <td><code>str</code>. If the <code>feature_biotype</code> is <code>"gene"</code> then this this MUST be the human-readable ENSEMBL gene name assigned to the <code>feature_id</code>. If the <code>feature_biotype</code> is <code>"spike-in"</code> then this MUST the ERCC Spike-In identifier appended with <code>" spike-in control"</code>.
         </td>
     </tr>
 </tbody></table>
 <br>
 
+### feature_reference
+
+<table><tbody>
+    <tr>
+      <th>Key</th>
+      <td>feature_reference</td>
+    </tr>
+    <tr>
+      <th>Annotator</th>
+      <td>Data Portal</td>
+    </tr>
+    <tr>
+      <th>Value</th>
+        <td><code>str</code>. This MUST indicate the reference organism of genes and MUST be <code>NCBITaxon:9606</code> for <i>Homo sapiens</i>, <code>NCBITaxon:10090</code> for <i>Mus musculus</i>, or <code>NCBITaxon:2697049</code> for  SARS-CoV-2.
```

Comment:
What's the instruction for the Portal if the feature is a spike-in? `ERCC`? `synthetic control`?

Reply:
This also needs an **If the feature_biotype is "gene"** conditional. Do we need a value if the feature is a spike-in?

Reply:
One possibility -  https://www.ncbi.nlm.nih.gov/nuccore/KC702166.1 indicates that it's _source organism_ is [_synthetic construct_](https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?id=32630) or:

Taxonomy ID: 32630 (for references in articles please use NCBI:txid32630)

Reply:
And that would fit with our use of NCBITaxon for the other cases.

Reply:
I like the suggestion of NCBITaxon:32630 for a synthetic spike-in. 👍

Reply:
Also _This must be the reference organism for ~~genes~~ features_ ... ?

Reply:
See Table 1 in https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4978944/. 

Looking at the spike-in examples in more detail - there are three DNA sources. Syn and Bsub are both synthetic and Mjan is an exception. 

For example - 
https://www.ncbi.nlm.nih.gov/nuccore/DQ516784

SOURCE      Methanocaldococcus jannaschii
  ORGANISM  Methanocaldococcus jannaschii

I still think that the proposed solution is a better approach than inventing another keyword.

Reply:
Agreed!